### ISSUE TITLE
Mobile View: Subscription form visibility

### ISSUE THREAD

**Issue by mk4111** [ayooo-team/landing-page#38]
###### Comment number 2 from Client's email (dated 24/03/2016)
_"The subscription form is not visible on the mobile version (I have an Iphone 6s)"_

**Comment by minaorangina** [comment]
Google form swapped out in favour of custom form, which does not require scrolling

**Comment by minaorangina** [comment]
Note to @PAOLOPRIAMODAMU:

We first implemented a Mailchimp form.  However we realised the users would be forced to confirm their subscription via email, which you strongly said you did not want.  At the time Google forms seemed to be the best alternative to satisfy your needs, as it would also allow subscriber information to go to a Google spreadsheet, for use with Mailchimp.

We have now created a custom form instead.  Subscriber information can be downloaded when you go to ayooo.herokuapp.com/admin.  Username and password will be emailed to you.
